### ISSUE TITLE
Travis deployment using different theme

### ISSUE THREAD

**Issue by ptheywood** [RSE-Sheffield/old_team_site_nikola#101]
After merging #100, the automatic nikola rebuild and deploy via travis has produced a different looking website to a local nikola build:

I.e. 
![image](https://user-images.githubusercontent.com/628937/46012816-4a726700-c0c2-11e8-9991-4d6069c3782b.png)

Rather than the expected local build result:

![image](https://user-images.githubusercontent.com/628937/46012989-b9e85680-c0c2-11e8-85e8-1dd308bb17ce.png)

Build log at https://travis-ci.org/RSE-Sheffield/RSE-Sheffield.github.io/builds/432908293

**Comment by ptheywood** [comment]
This appears to have been caused by an update to nikola (or nikola[extras]) (now version 8.0.0, from 7.8.15 on my local build), resulting in the bootstrap3 theme not being available:

> [2018-09-25T11:49:01Z] WARNING: Nikola: Cannot load theme "bootstrap3", using 'bootblog4' instead.
https://travis-ci.org/RSE-Sheffield/RSE-Sheffield.github.io/builds/432908293#L610

This has also raised several new warnings regarding deprecated features:

```
[2018-09-25T12:03:00Z] WARNING: Nikola: The UNSLUGIFY_TITLES setting was renamed to FILE_METADATA_UNSLUGIFY_TITLES.
[2018-09-25T12:03:00Z] WARNING: Nikola: The sections feature has been removed and its functionality has been merged into categories.
[2018-09-25T12:03:00Z] WARNING: Nikola: For more information on how to migrate, please read: https://getnikola.com/blog/upgrading-to-nikola-v8.html#sections-were-replaced-by-categories
[2018-09-25T12:03:00Z] WARNING: Nikola: Cannot load theme "bootstrap3", using 'bootblog4' instead.
[2018-09-25T12:03:00Z] WARNING: Nikola: The post "posts/philtooley_intro.md" uses the "mathjax" tag.
[2018-09-25T12:03:00Z] WARNING: Nikola: It is suggested that you convert special tags to metadata and set USE_TAG_METADATA to False. You can use the upgrade_metadata_v8 command plugin for conversion (install with: nikola plugin -i upgrade_metadata_v8). Change the WARN_ABOUT_TAG_METADATA configuration to disable this warning.
..[2018-09-25T12:03:00Z] WARNING: Nikola: The post "pages/index.rst" uses the "mathjax" tag.
[2018-09-25T12:03:00Z] WARNING: Nikola: It is suggested that you convert special tags to metadata and set USE_TAG_METADATA to False. You can use the upgrade_metadata_v8 command plugin for conversion (install with: nikola plugin -i upgrade_metadata_v8). Change the WARN_ABOUT_TAG_METADATA configuration to disable this warning.
[2018-09-25T12:03:02Z] WARNING: Nikola: Unknown path request of kind: section_index
[2018-09-25T12:03:02Z] WARNING: Nikola: Unknown path request of kind: section_index
[2018-09-25T12:03:02Z] WARNING: Nikola: Unknown path request of kind: section_index
[2018-09-25T12:03:02Z] WARNING: Nikola: Unknown path request of kind: section_index
```

**Comment by ptheywood** [comment]
Resolved by #103
